music_assistant/providers/snapcast/__init__.py

```diff
@@ -595,6 +595,7 @@ async def _streamer() -> None:
                 stream_path = stream.path
             if not stream.path:
                 stream_path = "tcp://" + stream._stream["uri"]["host"]
+            stream_path = stream_path.replace("0.0.0.0", self._snapcast_server_host)
```

Comment:
Isn't it better to construct the stream path right with the Snapcast server host in the line above? Or is the stream path sent from Snapcast server this wrong way?